### ISSUE TITLE
Any ideas about the structure of unsupervised SUM-FCN

### ISSUE THREAD

**Issue by chenbohua3** [weirme/FCSN#9]
After reading chapter 3.3 in FCSN several times, I can not figure out what exactly structure of the unsupervised part. Is that mean:
1. select Y frames:  choose the top Y socres features with dimension: 
     `batch * 2 * Y `
2. apply a 1*1 conv to decode features above to reconstruct their orginal feature  representations:
    `batch * 2 * Y  -> batch * 10 * Y (shape of the output of conv8)`
3. merge the input frame-level feature vectors of thess selected Y frames using skip connection:
    `batch * 1024 * Y  -> batch * 10 * Y ` 
    and then added by the output of step 2
4. obtain final reconstructed features of the Y frames:
    `batch * 10 * Y  -> batch * 1024 * Y`

**Comment by weirme** [comment]
I agree with your understanding in 1, 2, 4, and can we directly add the output of `conv8` and step2 in 3? Have you experimented with it?

**Comment by jiaweichuang** [comment]
Hello,have you implemented this unsupervised part?Can you tell me?
